### PR TITLE
Fixed and stabilized tests

### PR DESCRIPTION
**Pull request by roblaszczak**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,24 +5,105 @@ version: '3'
...
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9091,PLAINTEXT_HOST://localhost:29091
+      KAFKA_LOG4J_LOGGERS: "kafka.authorizer.logger=INFO"
       KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
-      KAFKA_NUM_PARTITIONS: 8
+      KAFKA_BROKER_RACK: "r1"
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 3
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
```
Comment (m110) [inline docker-compose.yml]: Perhaps the common envs could be moved to a separate file and included with `env_file`.

File: pkg/kafka/subscriber.go [hunk 2]
```diff
@@ -217,18 +237,13 @@ func (s *Subscriber) consumeGroupMessages(
...
-		cancel()
-	}()
-
 	// Start a new consumer group
 	group, err := sarama.NewConsumerGroupFromClient(s.config.ConsumerGroup, client)
 	if err != nil {
 		return nil, errors.Wrap(err, "cannot create consumer group client")
 	}
 	go func() {
+		// todo - better close?
```
Comment (m110) [inline pkg/kafka/subscriber.go]: Is this todo for later?

Comment (roblaszczak) [reply pkg/kafka/subscriber.go]: yup

File: docker-compose.yml (hunk 1 above)
Comment (roblaszczak) [reply docker-compose.yml]: 21b567f

File: pkg/kafka/subscriber.go (hunk 2 above)
Comment (roblaszczak) [reply pkg/kafka/subscriber.go]: but basically obsolete, removed

**Pull request merged**